internal/app/singularity/push.go

```diff
@@ -137,7 +137,7 @@ func LibraryPush(ctx context.Context, pushSpec LibraryPushSpec, libraryConfig *c
 		}
 	}(time.Now())
 
-	if resp, err = libraryClient.UploadImage(ctx, f, r.Host+r.Path, arch, r.Tags, pushSpec.Description, progressBar); err != nil {
+	if resp, err = libraryClient.UploadImage(ctx, f, r.Path, arch, r.Tags, pushSpec.Description, progressBar); err != nil {
```

Comment:
```
 $ singularity -d push -U ubuntu_20.04.sif library://library.staging.sylabs.io/dtrudg-sylabs-2/test/ubuntu:bob
 ```
 
 Now quitely pushes to the library of the currently selected remote... ignoring the host in the URI.
 
 ```
 2022/04/06 08:19:06 http2: Transport encoding header ":authority" = "library.sylabs.io"
2022/04/06 08:19:06 http2: Transport encoding header ":method" = "GET"
2022/04/06 08:19:06 http2: Transport encoding header ":path" = "/v1/entities/dtrudg-sylabs-2"
2022/04/06 08:19:06 http2: Transport encoding header ":scheme" = "https"
 ```

Reply:
Thanks @dtrudg, I'll ensure that path errors out as suggested.